runner/src/server/forms/report-a-terrorist.json

```diff
@@ -268,5 +283,23 @@
         ]
       }
     }
+  ],
+  "footer": [
+    {
+      "href": "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice",
+      "text": "Privacy test"
+    },
+    {
+      "href": "https://www.google.com",
+      "text": "Cookies test"
+    },
+    {
+      "href": "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice",
+      "text": "Accessibility Statement test"
+    },
+    {
+      "href": "https://www.gov.uk/government/publications/community-ownership-fund-privacy-notice/community-ownership-fund-privacy-notice",
+      "text": "Contact Us test"
+    }
   ]
```

Comment:
Should probably be obfuscating test data - but not specific to your PR so just ignore